src/firecracker/src/main.rs

```diff
@@ -149,124 +153,114 @@ fn main_exec() -> Result<(), MainError> {
 
     let http_max_payload_size_str = HTTP_MAX_PAYLOAD_SIZE.to_string();
 
-    let mut arg_parser = ArgParser::new()
-        .arg(
-            Argument::new("api-sock")
-                .takes_value(true)
-                .default_value(DEFAULT_API_SOCK_PATH)
-                .help("Path to unix domain socket used by the API."),
-        )
-        .arg(
-            Argument::new("id")
-                .takes_value(true)
-                .default_value(DEFAULT_INSTANCE_ID)
-                .help("MicroVM unique identifier."),
-        )
-        .arg(
-            Argument::new("seccomp-filter")
-                .takes_value(true)
-                .forbids(vec!["no-seccomp"])
-                .help(
-                    "Optional parameter which allows specifying the path to a custom seccomp \
-                     filter. For advanced users.",
+    let mut arg_parser =
+        ArgParser::new()
+            .arg(
+                Argument::new("api-sock")
+                    .takes_value(true)
+                    .default_value(DEFAULT_API_SOCK_PATH)
+                    .help("Path to unix domain socket used by the API."),
+            )
+            .arg(
```

Comment:
I have run `cargo fmt` in every commit, it seems the tool only felt it necessary to do it in this commit.

Reply:
Yeah it can be worked around, but moving forward, it would be good to split formatting changes from functional changes. At least the ones that span more than a few lines.